meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -1,10 +1,13 @@
 CONFIG_SPL=y
+# MAX SIZE = D000(SPL+dtb) + 2060(csf) - 400(fit signing data)
+CONFIG_SPL_SIZE_LIMIT=0xEC60
 CONFIG_SPL_SEPARATE_BSS=y
 CONFIG_SPL_LDSCRIPT="arch/arm/mach-omap2/u-boot-spl.lds"
 CONFIG_SPL_TEXT_BASE=0x00908000
-CONFIG_SPL_FIT_SIGNATURE=y
-CONFIG_SPL_FIT_SIGNATURE_STRICT=y
+CONFIG_SPL_FIT_SIGNATURE=n
+CONFIG_SPL_FIT_SIGNATURE_STRICT=n
 CONFIG_SPL_LOAD_FIT=y
+CONFIG_SPL_RSA=y
```

Comment:
So why we need RSA then?

Reply:
The only settings I'm changing in the lmp.cfg are 2 that are disabled strictly for size optimization.  I left this enabled so that as soon as we can re-enable `CONFIG_SPL_FIT_SIGNATURE` this would still be in the build.